crosschain-resolver/contracts/L1Resolver.sol

```diff
@@ -210,4 +211,12 @@ contract L1Resolver is EVMFetchTarget {
     ) public pure returns (bytes memory) {
         return abi.encode(values[1]);
     }
+
+    function supportsInterface(
+        bytes4 interfaceId
+    ) public pure returns (bool) {
+        return
+            interfaceId == type(IExtendedResolver).interfaceId ||
+            interfaceId == type(ITargetResolver).interfaceId;
```

Comment:
This needs to implement `ERC165` and  call the parent implementation.

Reply:
Looks like commit I branched out was not the latest. Fixed in the latest commit